sound/soc/sof/pcm.c

```diff
@@ -157,7 +157,7 @@ static int sof_pcm_hw_params(struct snd_pcm_substream *substream,
 	pcm.params.buffer_fmt = SOF_IPC_BUFFER_INTERLEAVED;
 	pcm.params.rate = params_rate(params);
 	pcm.params.channels = params_channels(params);
-	pcm.params.host_period_bytes = params_period_bytes(params);
+	pcm.params.host_period_bytes = params_period_size(params);
```

Comment:
This is wrong?

Reply:
The original one returns 0, the new one gives proper period size in # frames - as expected and documented by ALSA.